### PR TITLE
feat: new vault and syrup pool

### PR DESCRIPTION
**Pull request by ChefBingbong**
---

### Detailed summary
- Added `ASCEND` as a new strategy in the `types.ts` file.
- Updated the `strategy.ts` file to include a translation for the `ASCEND` strategy.
- Added the translation for `Ascend` in the `translations.json` file.
- Added a new pool configuration with `ASCEND` strategy in the `pools/56.ts` file.
- Added a new vault configuration with `ASCEND` strategy in the `vaults/eth.ts` file.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: packages/pools/src/constants/pools/56.ts [hunk 1]
```diff
@@ -13,6 +13,16 @@ export const livePools: SerializedPool[] = [
...
     isFinished: false,
   },
+  {
+    sousId: 367,
+    stakingToken: bscTokens.cake,
+    earningToken: bscTokens.irl,
+    contractAddress: '0x41cD0Fad28F8531De22617959bc943F1B3E12Bd8',
+    poolCategory: PoolCategory.CORE,
+    tokenPerBlock: '0.0694444444444445',
+    isFinished: false,
```
Comment (chefjackson) [inline packages/pools/src/constants/pools/56.ts:23]: remove the `isFinished` field

**Review by ChefJerry — APPROVED** [review]

File: packages/position-managers/src/types.ts [hunk 2]
```diff
@@ -17,6 +17,7 @@ export enum Strategy {
   PASSIVE,
   PEGGED,
   ALO,
+  ASCEND,
```
Comment (chefjackson) [inline packages/position-managers/src/types.ts:20]: we need to have new strategy name as well?

**Review by chefjackson — CHANGES_REQUESTED** [review]
please add name for the new startegy

File: packages/position-managers/src/types.ts (hunk 2 above)
Comment (ChefBingbong) [reply packages/position-managers/src/types.ts:20]: yeah this vault is exception. if u read messages in chat you will see that theres new strategy for this one

File: packages/pools/src/constants/pools/56.ts (hunk 1 above)
Comment (ChefBingbong) [reply packages/pools/src/constants/pools/56.ts:23]: removing

**Review by chefjackson — APPROVED** [review]

**Pull request merged**